_partials/_migrate_post_schema_caggs_etal.md

```diff
@@ -0,0 +1,165 @@
+    ```
+
+1.  Get a list of your existing continuous aggregate definitions:
+
+    ```sql
+    SELECT view_name, view_definition FROM timescaledb_information.continuous_aggregates;
+    ```
+
+    This query returns the names and definitions for all your continuous
+    aggregates. For example:
+
+    ```sql
+    view_name       |                                            view_definition
+    ----------------+--------------------------------------------------------------------------------------------------------
+    avg_fill_levels |  SELECT round(avg(fill_measurements.fill_level), 2) AS avg_fill_level,                                +
+                    |     time_bucket('01:00:00'::interval, fill_measurements."time") AS bucket,                            +
+                    |     fill_measurements.sensor_id                                                                       +
+                    |     FROM fill_measurements                                                                            +
+                    |     GROUP BY (time_bucket('01:00:00'::interval, fill_measurements."time")), fill_measurements.sensor_id;
+    (1 row)
+    ```
+
+1.  Connect to your Timescale database:
+
+    ```bash
+    psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
+    ```
+
+1.  Recreate each continuous aggregate definition:
+
+    ```sql
+    CREATE MATERIALIZED VIEW <VIEW_NAME>
+    WITH (timescaledb.continuous) AS
+    <VIEW_DEFINITION>
+    ```
+
+</Procedure>
+
+## Recreate policies
+
+By default, policies aren't migrated when you transfer your schema and data
+separately. Recreate them on your Timescale database.
+
+<Procedure>
+
+### Recreating policies
+
+1.  Connect to your source database:
+
+    ```sql
+    psql "postgres://<SOURCE_DB_USERNAME>:<SOURCE_DB_PASSWORD>@<SOURCE_DB_HOST>:<SOURCE_DB_PORT>/<SOURCE_DB_NAME>?sslmode=require"
+    ```
+
+1.  Get a list of your existing policies. This query returns a list of all your
+    policies, including continuous aggregate refresh policies, retention
+    policies, compression policies, and reorder policies:
+
+    ```sql
+    SELECT application_name, schedule_interval, retry_period,
+        config, hypertable_name
+        FROM timescaledb_information.jobs WHERE owner = '<SOURCE_DB_USERNAME>';
+    ```
+
+1.  Connect to your Timescale database:
+
+    ```sql
+    psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
+    ```
+
+1.  Recreate each policy. For more information about recreating policies, see
+    the sections on [continuous-aggregate refresh policies][cagg-policy],
+    [retention policies][retention-policy], [compression
+    policies][compression-policy], and [reorder policies][reorder-policy].
+
+</Procedure>
+
+## Update table statistics
+
+Update your table statistics by running [`ANALYZE`][analyze] on your entire
+dataset:
+
+```sql
+ANALYZE;
+```
```

Comment:
Warn that this might take some time depending on the amount of data.

Reply:
done